### ISSUE TITLE
Usability: NVDA  is not reciting the "Name" and "shortcut key " for the view bar buttons in the visual studio code editor.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17225]
VS Code Version: 1.6.1
OS Version:1607
NVDA version:-2016.2.1

**Steps to Reproduce:**
Prerequisite: NVDA need to be opened.
1.Open visual studio code.
2.Using "Ctrl+ Shift+ E", key navigate to the Explorer button.
3.Using "Ctrl+ Shift+ F", key navigate to the Search button.
4.Using "Ctrl+ Shift+ G", key navigate to the Git button.
5.Using "Ctrl+ Shift+ D", key navigate to the Debug button.
6.Using "Ctrl+ Shift+ X", key navigate to the "Extension" button. 

**Expected Result:**
NVDA  should announce the name and the shortcut key for  the view bar buttons correctly.

**Actual Result:**
NVDA  is not announcing anything for  the view bar buttons present in the vs code editor.
Note: Same issue is observed to all the view bar buttons(Explorer, Search ,Git, Debug, Extension).

**User Impact:**
Screen Reader user doesn't get to know that the controls corresponds to which View bar.

![inspectinformation_visualstudiocodeeditor_search_viewbarbuttons](https://cloud.githubusercontent.com/assets/22956130/21212343/b49cd180-c2b1-11e6-8291-2eb7ea0bf11d.jpg)

![nvda_speechviewerinformation_vscodeeditor_viewbaroption](https://cloud.githubusercontent.com/assets/22956130/21212345/babad60c-c2b1-11e6-8e25-453b7ae3468d.jpg)

**Comment by bpasero** [comment]
@WiproPSAccTest not sure about this one, when you e.g. press Ctrl+Shift+E, we move focus into the explorer so the actual view button is not focussed. When you use the keyboard to navigate to the view buttons you can hear NVDA read out correctly label and keybinding. I think we should not change the behaviour of moving focus INTO the view. I would expect that NVDA reads whatever is focussed IN the view then.

**Comment by chrisdias** [comment]
Accessibility Triage: Agreed with @bpasero on not changing focus. But, can we announce the Viewlet  title when focus changes?

**Comment by bpasero** [comment]
@chrisdias only if we add the label of the viewlet as prefix to each and every thing that can have focus inside the viewlet. We could also abuse the aria alert to announce the change. 

I find this issue a bit artificial because we have not heard complaints about this so far.

**Comment by WiproPSAccTest** [comment]
@bpasero @chrisdias It is good to have that NVDA should announce the name for the "view bar" options. currently NVDA is announcing the options present in the "side bar" options without specifying the name of the  "view bar" option associated with it. For a blind user it is difficult to know which side bar option is associated with which view bar option.

**Comment by bpasero** [comment]
Here is what happens today when you use these keys to switch viewlets and I think the context is very good and does not need to be more verbose:
- Explorer: "Files Explorer, <selected file>"
- Search: "Search, you are currently on a text field"
- Git: "Git: Type commit..."
- Debug: "Start debugging"
- Extensions: "Search extensions in marketplace"

Let me know if I am missing something here.

**Comment by chrisdias** [comment]
If I have Open Editors turned on it says "Open Editors Tree View".  I know that when sitting with Florian, he also had trouble understanding what "Open Editors" meant so perhaps we can special case Explorer to be a little more verbose.

**Comment by isidorn** [comment]
After consulting our docs I changed it to "Open Editors: List of Active Files". Feedback on better wording is welcome
